bc/core/utils/context_processors.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+
+from bc.core.models import BannerConfig
+
+
+def banner_settings(request):
+    active_banner = BannerConfig.objects.filter(is_active=True).first()
+    return {
+        "HEADER_BANNER_ENABLED": bool(active_banner),
+        "HEADER_BANNER_TITLE": (active_banner.title if active_banner else ""),
+        "HEADER_BANNER_TEXT": (active_banner.text if active_banner else ""),
+        "HEADER_BANNER_BUTTON_TEXT": (
+            active_banner.button_text if active_banner else ""
+        ),
+        "HEADER_BANNER_BUTTON_LINK": (
+            active_banner.button_link if active_banner else ""
+        ),
```

Comment:
I don't know if this is any better, but it feels a bit more clear to me than all the if/else's: 

```suggestion
        "HEADER_BANNER_TITLE": getattr(active_banner, "title", ""),
        "HEADER_BANNER_TEXT": getattr(active_banner, "text", ""),
        "HEADER_BANNER_BUTTON_TEXT": getattr(active_banner, "button_text", ""),
        "HEADER_BANNER_BUTTON_LINK": getattr(active_banner, "button_link", ""),
```

But not worth changing and dealing with Black and all that. :)